src/core/tritonserver.cc

```diff
@@ -1523,6 +1539,7 @@ TRITONSERVER_ServerNew(
   lserver->SetStrictReadinessEnabled(loptions->StrictReadiness());
   lserver->SetExitTimeoutSeconds(loptions->ExitTimeout());
   lserver->SetBackendCmdlineConfig(loptions->BackendCmdlineConfigMap());
+  ni::AsyncWorkQueue::Initialize(loptions->BufferManagerThreadCount());
```

Comment:
This should be in server.cc::Init function. At the beginning just before the pinned memory manager.